packages/swingset-runner/demo/terminateVat/bootstrap.js

```diff
@@ -0,0 +1,35 @@
+/* global harden */
+
+import { E } from '@agoric/eventual-send';
+
+export function buildRootObject() {
+  let counter = 46;
+  const self = harden({
+    async bootstrap(vats, devices) {
+      const vatMaker = E(vats.vatAdmin).createVatAdminService(devices.vatAdmin);
+      const dude = await E(vatMaker).createVatByName('dude');
+      const succBefore = await E(dude.root).dude(true);
+      console.log(`success result ${succBefore}`);
+      try {
+        const failBefore = await E(dude.root).dude(false);
+        console.log(`failure path should not yield ${failBefore}`);
+      } catch (e) {
+        console.log(`failure result ${e}`);
+      }
+      await E(dude.root).elsewhere(self); // this will notify
+      E(dude.root).elsewhere(self); // this will not
```

Comment:
This would make a nice unit test in swingset proper, to confirm that previously-queued messages (and previously-queued notifications) are dropped properly. Could you see if you can port this back into swingset?

Reply:
I struggled unsuccessfully for a half hour trying to figure out how to do just that, and finally decided I'd just commit the code in the interest of moving things along, figuring that I'd ask you about it in the morning (or, as it turns out, have you notice it and complain about it :-)

In the swingset runner demo I can verify the behavior by inspecting the verbose mode log, but I can't for the life of me figure out how to expose it to the unit test assert framework, since at that point it's a check for the absence of something in a log that doesn't exist.

Reply:
ok, I'll take a stab at porting the test, will let you know what I come up with

Reply:
It's the curious incident of the dog in the night-time.

Reply:
ok #1497 adds a fairly comprehensive test for you to merge into this branch

Reply:
> I'll merge this since it's more comprehensive, but I still don't see how it tests for the case I was probing with my swingset runner demo. All it checks is that the notify was taken off the queue, which it will be either way.

oh, maybe I misread your demo.. let's chat tomorrow about what addtional checks we might add to the test